lib/http.js

```diff
@@ -193,6 +193,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     // not cookies
     cors(),
     validate('bedrock-edv-storage.document'),
+    helpers.verifyDigestHeaderValue,
```

Comment:
We should move the digest checking above the `validate` calls to ensure we're checking that what was sent was signed before we even start looking at it.

Reply:
Updated here 67cbad2